packages/core/test/feedback/FeedbackButton.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { fireEvent, render, waitFor } from '@testing-library/react-native';
+import * as React from 'react';
+
+import { FeedbackButton } from '../../src/js/feedback/FeedbackButton';
+import type { FeedbackButtonProps, FeedbackButtonStyles } from '../../src/js/feedback/FeedbackWidget.types';
+import { showFeedbackWidget } from '../../src/js/feedback/FeedbackWidgetManager';
+
+jest.mock('../../src/js/feedback/FeedbackWidgetManager', () => ({
+  ...jest.requireActual('../../src/js/feedback/FeedbackWidgetManager'),
+  showFeedbackWidget: jest.fn(),
+}));
+
+const defaultProps: FeedbackButtonProps = {
```

Comment:
L: `customTextProps` would be more expressive; with the defaultProps, I wasn't sure what it means at first.

Reply:
Good point Krystof 👍 
Updated with fc01d8c2837c45f42338e99ac6720248fd8e5f97